### PR TITLE
Refactor Abstraxion to fetch config

### PR DESCRIPTION
**Pull request by BurntVal**

### CODE REVIEW

File: packages/abstraxion/src/components/AbstraxionSignin/index.tsx [hunk 1]
```diff
@@ -87,7 +96,7 @@ export function AbstraxionSignin(): JSX.Element {
     }
     urlParams.set("redirect_uri", currentUrl);
     const queryString = urlParams.toString(); // Convert URLSearchParams to string
-    window.location.href = `${dashboardUrl}?${queryString}`;
+    window.location.href = `https://${dashUrl}?${queryString}`;
```
Comment (justinbarry) [inline packages/abstraxion/src/components/AbstraxionSignin/index.tsx:99]: The dashboard URL might be local, should include the protocol in the passed var

File: packages/constants/src/index.ts [hunk 2]
```diff
@@ -88,3 +85,26 @@ export const testChainInfo: ChainInfo = {
   chainId: "xion-local-testnet-1",
   chainName: "Xion Testnet Local",
 };
+
+// If mainnet chain-id/network changes be sure to update here.
+const DASHBOARD_URLS = {
+  "xion-mainnet-1": "dashboard.burnt.com",
+  "xion-testnet-1": "testnet-dashboard.burnt.com",
```
Comment (justinbarry) [inline packages/constants/src/index.ts:92]:
```suggestion
  "xion-testnet-1": "testnet.dashboard.burnt.com",
```

**Review by justinbarry — CHANGES_REQUESTED** [review]
Nice work! Requested 2 changes.

File: packages/constants/src/index.ts (hunk 2 above)
Comment (BurntVal) [reply packages/constants/src/index.ts:92]: I can make this change but I was under the impression that we wanted to stay away from using too many "." and thus the reason for the hyphenated sub-domain

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**